examples/post.janet

```diff
@@ -0,0 +1,21 @@
+(import janet-html)
+
+(def server-error-response
```

Comment:
Adding some comments explaining what the code is doing and **why** would be very helpful, since these examples are meant to be informative to new users.